views/account/settings.hbs

```diff
@@ -32,14 +32,14 @@
 
                 <div class="form-group">
                     <label for="email">{{$t "global.label.emailAddress" }}</label>
-                    <input id="email" type="email" class="form-control" name="email" value="{{ currentUser.email }}">
+                    <input id="email" type="email" class="form-control" name="email" required value="{{ currentUser.email }}">
                 </div>
 
                 <div class="form-group">
                     <label for="settings_current_password">{{$t "account.global.label.currentPassword" }} <sup>*</sup></label>
                     <div class="input-group show_hide_password">
-                        <input type="password" autocomplete="current-password" id="settings_current_password" data-testid="settings_password_current" name="password" class="form-control"
-                        placeholder="***************">
+                        <input type="password" autocomplete="false" id="settings_current_password" data-testid="settings_password_current" name="password" class="form-control"
```

Comment:
Why this value is false instead "off"?

Reply:
I found that off should be on form, on inputs it should be on false, for more chrome versions support